HISTORY.md

```diff
@@ -14,6 +14,7 @@
 * Add new option allow_stall passed during instance creation of WriteBufferManager. When allow_stall is set, WriteBufferManager will stall all writers shared across multiple DBs and columns if memory usage goes beyond specified WriteBufferManager::buffer_size (soft limit). Stall will be cleared when memory is freed after flush and memory usage goes down below buffer_size.
 * Allow `CompactionFilter`s to apply in more table file creation scenarios such as flush and recovery. For compatibility, `CompactionFilter`s by default apply during compaction. Users can customize this behavior by overriding `CompactionFilterFactory::ShouldFilterTableFileCreation()`.
 * Added more fields to FilterBuildingContext with LSM details, for custom filter policies that vary behavior based on where they are in the LSM-tree.
+* Add CompactionOptionsFIFO.age_for_warm, which allows RocksDB to move old files to warm tier in FIFO compactions. Note that file temperature is still an experimental feature.
```

Comment:
Yes we can do that. But I think it is beyond the scope of the PR, as this term is in other places. We need to change them all together as a separate PR if we want to.

Reply:
> Yes we can do that. ... if we want to.

@siying Thanks. I don't know if the conclusion is we'll do it, or we are still deciding, so will add a couple more reasons here.

- The original LSM paper and more define temperature purely as density of access: "Following Copeland et al. [6], we define the temperature of a body of data as H/S". (`H` is IOPs, `S` is storage space).
- We might measure pure temperature in the future, which would cause naming conflict.